In [8]:
import math

class Solution:
    def solution(self, Nums):
        money,person = map(int, Nums.split())
        
        result1 = money/person
        result2 = money%person
        print(int(result1))
        print(result2)
        
        
        
inputs = input()        
test = Solution()
test.solution(inputs)

10000000 123
81300
100


In [16]:
import math


inputs = input()        

if inputs != None:
    money,person = map(int, inputs.split())

    result1 = money/person
    result2 = money%person
    print(int(result1))
    print(int(result2))

120 10
12
0


In [17]:
n, m = map(int, input().split())
print(n//m)
print(n%m)

120 10
12
0


//

2048 // 123 -> 16

33 // 5 -> 6

소수점 이하의 수를 모두 버리고 몫만 나타낼 때 ‘//’ 연산자를 사용한다.



출처: https://includestdio.tistory.com/16 [includestdio]

/

2048 / 123 -> 16.650406504065042

33 / 5 -> 6.6

정확히 나누어 떨어지지 않는 경우, 파이썬에서는 소수점 15자리 까지 표현한다.


<!-- /

>>> 2048 / 123

16.650406504065042

정확히 나누어 떨어지지 않는 경우, 파이썬에서는 소수점 15자리 까지 표현한다. -->